docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -71,38 +105,54 @@ CPAN].
 
 [ZNMSTR account at CPAN]: http://search.cpan.org/~znmstr/
 
-## 7. Verify that the module builds and all tests pass
+## 8. Verify that the module builds and all tests pass
+
+Find the list of supported Perl versions and locales in the [declaration of
+prerequisites]. Then find the latest point-release of each supported Perl
+version in the table of [latest releases in each branch of Perl].
+
+Make sure you've set up perlbrew and installed the latest point-release of each
+supported Perl version for perlbrew.
+
+    perlbrew init
+    perlbrew install perl-5.14.4  # etc
+
+Perform the following for each component.
 
 Verify that the module builds and all tests pass with the latest point release
 for every supported major Perl version and for every supported system locale.
-This can be done quite easily with something like this command for each locale:
+For each supported locale and the set of supported point-releases, modify and
+run the folloing command:
 
-    LC_ALL=en.UTF-8 LC_MESSAGES=en.UTF-8 LC_NUMERIC=en.UTF-8 LANG=en.UTF-8 perlbrew exec --with 5.14.4,5.16.3,5.18.4,5.20.1 '( git clean -dfx && perl Makefile.PL && make ) >& /dev/null && prove -bQ'
+    LC_ALL=en.UTF-8 LC_MESSAGES=en.UTF-8 LC_NUMERIC=en.UTF-8 LANG=en.UTF-8 perlbrew exec --with 5.14.4,5.16.3,5.18.4 '( git clean -dfx && perl Makefile.PL && make ) >/dev/null && prove -bQ'
```

Comment:
--with 5.14.4,5.16.3,5.18.4,5.20.3,5.22.4,5.24.3

Reply:
Making this update now. In the long term I'm not sure about the entire perlbrew thing since we have Travis. They kind of solve the same problem.